client/post-editor/media-modal/detail/detail-item.jsx

```diff
@@ -75,15 +80,110 @@ class EditorMediaModalDetailItem extends Component {
 
 		return (
 			<Button
-				className={ classNames( 'editor-media-modal-detail__edit', className ) }
+				className="editor-media-modal-detail__edit"
```

Comment:
Yes, it's because we were only switching the `Edit Image` button depending on the width f the viewport. Now we have to handle other buttons as well, such as the new `Restore Original` button. I've added the `editor-bar` container to include these elements:

![image](https://cloud.githubusercontent.com/assets/77539/19786564/f9750786-9c74-11e6-9105-fe548abd8b95.png)